pegjs/postgresql.pegjs

```diff
@@ -3187,7 +3187,15 @@ table_name
 
 or_and_expr
 	= head:expr tail:(__ (KW_AND / KW_OR) __ expr)* {
-    // => binary_expr
+    /*
+    export type binary_expr = {
+      type: 'binary_expr';
+      operator: string;
+      left: string;
+      right: string;
```

Comment:
@DemonHa left and right property should be expr? and we have binary_expr before, do not change it here.

Reply:
the postgresql.ts before
```javascript
export type BINARY_OPERATORS = LOGIC_OPERATOR | 'OR' | 'AND' | multiplicative_operator | additive_operator
      | arithmetic_comparison_operator
      | 'IN' | 'NOT IN'
      | 'BETWEEN' | 'NOT BETWEEN'
      | 'IS' | 'IS NOT'
      | 'LIKE'
      | '@>' | '<@' | OPERATOR_CONCATENATION | DOUBLE_WELL_ARROW | WELL_ARROW | '?' | '?|' | '?&' | '#-'
    export interface binary_expr {
      type: 'binary_expr',
      operator: BINARY_OPERATORS,
      left: expr,
      right: expr
    }
```

Reply:
Ok, I see it now, but I think the typegen output file was modified by hand, I will change it!